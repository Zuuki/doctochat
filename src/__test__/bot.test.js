import React from 'react';
import ReactDOM from 'react-dom';
import ChatRoom from '../App';

import {render, cleanup} from "@testing-library/react"
import "@testing-library/jest-dom"

import renderer from 'react-test-renderer';

afterEach(cleanup)

it("Should renders without crashing", ()=>{
    const div = document.createElement("div");
    ReactDOM.render(<ChatRoom></ChatRoom>, div)
})

it('Should matches snapshot', ()=>{
    const tree = renderer.create(<ChatRoom></ChatRoom>).toJSON();
    expect(tree).toMatchSnapshot();
});