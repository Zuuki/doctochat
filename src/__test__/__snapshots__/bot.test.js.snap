// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should matches snapshot 1`] = `
<div
  className="App"
>
  <header>
    <img
      alt="Logo Doctolib"
      className="logo_doctolib"
      src="Doctolib_logo.png"
    />
    <h1>
       Chat Bot 
    </h1>
  </header>
  <section>
    <main>
      <div
        className="message sent"
      >
        <p>
          Bienvenue sur le Chat Bot Doctolib
        </p>
      </div>
      <div
        className="message sent"
      >
        <p>
          Voulez-vous:
        </p>
      </div>
      <div
        className="message sent"
      >
        <p>
          - 1. Exporter votre calendrier de rendez-vous sous le format .ics ?
        </p>
      </div>
      <div
        className="message sent"
      >
        <p>
          - 2. Chercher quel practicien vous devriez consulter selon vos sympt√¥mes ?
        </p>
      </div>
      <span />
    </main>
    <form
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ecrivez ici"
        value=""
      />
      <button
        disabled={true}
        type="submit"
      >
        sedundnes
      </button>
    </form>
  </section>
</div>
`;
